src/server/search/doc_accessors.cc

```diff
@@ -2,14 +2,21 @@
 // See LICENSE for licensing terms.
 //
 
+// GCC yields a spurious warning about uninitialized data in DocumentAccessor::StringList.
+#ifdef __GNUC__
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
+#endif
```

Comment:
I assume that if you never pop it it'll retain only for the compilation unit, I assume the best scope would be only the function itself if it works

Reply:
it does not. tried it for 30mins and have not understood the logic behind it.